addons/l10n_in/models/company.py

```diff
@@ -84,3 +84,17 @@ def _check_l10n_in_pan(self):
     def action_update_state_as_per_gstin(self):
         self.ensure_one()
         self.partner_id.action_update_state_as_per_gstin()
+
+    def _get_token_list(self):
```

Comment:
```suggestion
    def _get_l10n_in_api_list(self):
```
better to keep `l10n_in` in method name to avoid conflicts with main modules